packages/runtime/runtime-utils/src/snapshotUtils.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { ISnapshotTree } from "@fluidframework/protocol-definitions";
+
+/**
+ * Utility function to check if any blobs under a snapshot tree is missing and if so, then return
+ * true if that is the case.
+ * @internal
+ * @param snapshotTree - snapshotTree to be evaluated for missing blobs.
+ * @param blobContents - blobContents of the snapshot.
+ */
+export function isSnapshotFetchRequiredForLoadingGroupId(
+	snapshotTree: ISnapshotTree,
+	blobContents: Map<string, ArrayBuffer>,
+): boolean {
+	return evaluateSnapshotTreeForMissingBlobs(snapshotTree, blobContents);
+}
+
+function evaluateSnapshotTreeForMissingBlobs(
```

Comment:
what is the value of this function? I.e. why it's not inlined into isSnapshotFetchRequiredForLoadingGroupId?